.github/workflows/tests/smoke_test_auth_cherry_picker.sh

```diff
@@ -0,0 +1,40 @@
+# Replace our build support files with stub ones
+cat << EOF > build-support/bin/cherry_pick/make_pr.sh
+#!/usr/bin/env bash
+echo "I would've made a PR!"
+# We exit 1 to test we still call the finish job
+exit 1
+EOF
+chmod +x build-support/bin/cherry_pick/make_pr.sh
+
+cat << EOF > build-support/bin/cherry_pick/helper.js
+class CherryPickHelper {
+    constructor({ octokit, context, core }) {}
+    async get_prereqs() {
+        return {
+            pr_num: 12345,
+            merge_commit: "ABCDEF12345",
+            milestones: ["2.16.x", "2.17.x"],
+        };
+  }
+
+    async cherry_pick_finished(merge_commit_sha, matrix_info) {
+        console.log(\`We finished: \${merge_commit_sha} \${matrix_info}\`);
```

Comment:
Printing `[object Object],[object Object]` seems like a misleading way of validating the length. Here's some alternatives:

Show the actual contents

```suggestion
        console.log(\`cherry_picked_finished: \${merge_commit_sha} \${JSON.stringify(matrix_info)}\`);
```

Validate the length explicitly:

```suggestion
        console.log(\`cherry_picked_finished: \${merge_commit_sha} \${matrix_info.length}\`);
```

Doing this as follow-up is fine by me.